yubikey-u2f-pam-plugin/auth-pam-u2f.c

```diff
@@ -755,7 +756,7 @@ pam_server(int fd, const char *service, int verb, const struct name_value_list *
      */
     if (!dlopen_pam(pam_so))
     {
-        fprintf(stderr, "AUTH-PAM: BACKGROUND: could not load PAM lib %s: %d\n", pam_so, dlerror());
+        fprintf(stderr, "AUTH-PAM: BACKGROUND: could not load PAM lib %s: %s\n", pam_so, dlerror());
```

Comment:
While this is technically correct, in practice we specifically want an integer representation of the error code to make looking it up easier.